app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -73,8 +72,8 @@ protected void onCreate(Bundle savedInstanceState) {
                 this,
                 IoniconsIcons.ion_android_upload).colorRes(R.color.white));
 
-        nameListsAdapter = new NameListsAdapter(this, noContent);
-        lists.setAdapter(nameListsAdapter);
+        nameListsAdapter = new NameListsAdapter(this,this, noContent);
```

Comment:
nit: We're going to end up changing this but leaving a reminder to space things like this when we get here.

```suggestion
        nameListsAdapter = new NameListsAdapter(this, this, noContent);
```